src/main/java/seedu/address/logic/commands/UntagCommand.java

```diff
@@ -0,0 +1,118 @@
+package seedu.address.logic.commands;
+
+import static java.util.Objects.requireNonNull;
+import static seedu.address.logic.parser.CliSyntax.PREFIX_TAG;
+import static seedu.address.model.Model.PREDICATE_SHOW_ALL_PERSONS;
+
+import java.util.HashSet;
+import java.util.List;
+import java.util.Set;
+import java.util.stream.Collectors;
+
+import seedu.address.commons.core.Messages;
+import seedu.address.commons.core.index.Index;
+import seedu.address.logic.commands.exceptions.CommandException;
+import seedu.address.model.Model;
+import seedu.address.model.person.Address;
+import seedu.address.model.person.Birthday;
+import seedu.address.model.person.Email;
+import seedu.address.model.person.Name;
+import seedu.address.model.person.Person;
+import seedu.address.model.person.Phone;
+import seedu.address.model.tag.Tag;
+
+public class UntagCommand extends EditCommand {
```

Comment:
Do you want to maybe look at Daniel's command to see how he did it? I'm concerned that the two command that both of you did may have very different implementations despite being very similar.

Reply:
My main concern with inheriting from edit directly was to reduce/prevent regression if/when edit is changed
but yes need to discuss this